### ISSUE TITLE
Is there an opposite of rewriteArrowSymbols?

### ISSUE THREAD

**Issue by jsatk** [scala-ide/scalariform#286]
Re: http://scala-ide.org/scalariform/#rewritearrowsymbols

There's been some discussion recently about unicode arrows being unsafe and preferring ascii arrows and having people use fonts with ligatures if they want fancy-looking arrows.

https://nrinaudo.github.io/scala-best-practices/tricky_behaviours/unicode_operators.html
https://github.com/scala/scala/pull/7540

I'd love to be able to enforce an option that rewrote all unicode arrows to ascii arrows. Lots of devs have this settting check in their IntelliJ. Would be nice to just automate it.

**Comment by dhorkin** [comment]
Yes, but it doesn't work. I just filed a bug for it: https://github.com/scala-ide/scalariform/issues/289
